dialog/fileitem.go

```diff
@@ -88,22 +105,41 @@ type fileItemRenderer struct {
 func (s fileItemRenderer) Layout(size fyne.Size) {
 	s.background.Resize(size)
 
-	iconAlign := (size.Width - fileIconSize) / 2
-	s.icon.Resize(fyne.NewSize(fileIconSize, fileIconSize))
-	s.icon.Move(fyne.NewPos(iconAlign, 0))
+	if s.item.picker.view == gridView {
+		s.icon.Resize(fyne.NewSize(fileIconSize, fileIconSize))
+		s.icon.Move(fyne.NewPos((size.Width-fileIconSize)/2, 0))
+
+		s.text.Alignment = fyne.TextAlignCenter
+		s.text.Resize(fyne.NewSize(size.Width, fileTextSize))
+		s.text.Move(fyne.NewPos(0, size.Height-s.text.MinSize().Height))
+	} else {
+		s.icon.Resize(fyne.NewSize(fileInlineIconSize, fileInlineIconSize))
+		s.icon.Move(fyne.NewPos(0, (size.Height-fileInlineIconSize)/2))
+
+		s.text.Alignment = fyne.TextAlignLeading
+		s.text.Resize(fyne.NewSize(size.Width, fileTextSize))
+		s.text.Move(fyne.NewPos(fileInlineIconSize, (size.Height-s.text.MinSize().Height)/2))
+	}
 
-	s.text.Resize(fyne.NewSize(size.Width, fileTextSize))
-	s.text.Move(fyne.NewPos(0, size.Height-fileTextSize-theme.Padding()*2))
 }
 
 func (s fileItemRenderer) MinSize() fyne.Size {
-	return fyne.NewSize(fileIconSize, fileIconSize+fileTextSize+theme.Padding())
+	var minsize fyne.Size
+	if s.item.picker.view == gridView {
+		minsize = fyne.NewSize(fileIconSize, fileIconSize+fileTextSize+theme.Padding())
+	} else {
+		minsize = fyne.NewSize(fileInlineIconSize+s.text.MinSize().Width, fileTextSize+theme.Padding()*4)
+	}
+	return minsize
```

Comment:
```suggestion
	if s.item.picker.view == gridView {
		return fyne.NewSize(fileIconSize, fileIconSize+fileTextSize+theme.Padding())
	}

	return fyne.NewSize(fileInlineIconSize+s.text.MinSize().Width, fileTextSize+theme.Padding()*4)
```
I think this is a preferred and cleaner way to do it.

Reply:
I wanted to do it that way, just it felt like we are making a slight separation by excluding the else statement.
So what do you?
Should i just make the change anyway as it's not really a problem?

Reply:
It is much cleaner the way I suggested. It is generally the preferred way to do it if looking through other parts of code here.